java/src/main/java/lesson01/README.md

```diff
@@ -101,7 +101,7 @@ Our `pom.xml` already imports Jaeger:
 <dependency>
     <groupId>com.uber.jaeger</groupId>
     <artifactId>jaeger-core</artifactId>
-    <version>0.26.0</version>
+    <version>0.27.0</version>
```

Comment:
we should change it to `io.jaegertracing`

Reply:
Oh, ok. I didn't know this package has been moved to io.jaegertracing:jaeger-core. The java package names have changed, so imports and fully qualified class references would have to be updated too. I can update this pull request with these changes as well.